tests/test_referenceObjectLoader.py

```diff
@@ -139,21 +135,10 @@ def testMakeMinimalSchema_fullCovariance(self):
         self.assertIn("pm_raErr", schema)
         self.assertIn("pm_ra_parallax_Cov", schema)
         self.assertIn("parallaxErr", schema)
-        self.assertEqual(schema['raErr'].asField().getUnits(), "rad")
+        self.assertEqual(schema['coord_raErr'].asField().getUnits(), "rad")
         self.assertEqual(schema['coord_ra_coord_dec_Cov'].asField().getUnits(), "rad rad")
-        self.assertEqual(schema['pm_raErr'].asField().getUnits(), "rad/yr")
-        self.assertEqual(schema['pm_dec_parallax_Cov'].asField().getUnits(), "rad/yr rad")
-
-        # coordErrDim is completely ignored for fullCoordinateCovariance=True
-        schema = ReferenceObjectLoader.makeMinimalSchema(filterNameList=["r"],
-                                                         fullCoordinateCovariance=True,
-                                                         addParallax=True,
-                                                         addProperMotion=True,
-                                                         coordErrDim=-1)
-        # Again, just test a sample of the fields.
-        self.assertIn("coord_raErr", schema)
-        self.assertIn("coord_ra_coord_dec_Cov", schema)
-        self.assertIn("pm_ra_parallax_Cov", schema)
```

Comment:
Because this was to test that `coordErrDim` was being ignored, which is no longer the case. Line 154 would have to be `self.assertNotIn("coord_raErr", schema)` to pass now.